company-profiles/infuseai.md

```diff
@@ -0,0 +1,30 @@
+# InfuseAI
+
+## Company blurb
+
+[InfuseAI](https://www.infuseai.io/), InfuseAI is a remote-friendly company based in Taiwan.
+Our product PrimeHub is a deep learning platform to streamline AI workflow and supercharge productivity for data scientists. If you are a dream practitioner who strives to accelerate the best AI platform to make the world fantastic, welcome to join InfuseAI - a tech-savvy, open-minded, remote-friendly company.
+
+## Company size
+
+Infuse is currently around 18 employees, distributed in Taiwan and Japan.
+
+## Remote status
+
+Most of engineering is remote, and a fair amount of sales and marketing are remote too. We use Zoom for face to face remote conversations, and we primarily use Slack for text-based conversations.
+
+## Region
+
+InfuseAI is currently hiring Worldwide.
+
+## Company technologies
+
+We used Gitlab and Github for Development. Clubhouse and Status Hero for issue tracking. Airtable, Docsend, DocuSign, GSuite, Miro, Notion, Slack, Zendesk, Zoom for Communication and Documentation. Lastly, we used Attendancebot and Recruitee for Human Resource Management.
+
+## Office locations
+
+InfuseAI is headquartered in Taipei, Taiwan.
+
+## How to apply
+
+Please apply through [our jobs listing page](https://jobs.infuseai.io/).
```

Comment:
Please find a place they host their jobs such as linkedin or change to a contact page.

Reply:
@KieranRobson  Done with the changes.